docs/SLSA-VERIFICATION.md

```
@@ -0,0 +1,121 @@
+- **Invalid signature**: The provenance file has been tampered with
+- **Expired certificate**: The signing certificate has expired (contact maintainers)
+
+**⚠️ DO NOT USE packages that fail verification.**
+
+## Provenance File Format
+
+The provenance file is in [in-toto](https://in-toto.io/) format with SLSA v1.0 predicate. It contains:
+
+- **Subject**: SHA256 hashes of all packages
+- **Predicate**: Build metadata including:
+  - Builder: `slsa-framework/slsa-github-generator`
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)